src/js/components/githubUsers/assignEpicCollaborators.tsx

```diff
@@ -75,7 +75,7 @@ const AssignEpicCollaboratorsModal = ({
     [selection],
   );
 
-  // When modal is canceled, reset row selection
+  // When modal is cancelled, reset row selection
```

Comment:
Oh geez, this led me down a "canceled vs cancelled" rabbit hole... Any opinions? 😂

Reply:
@jgerigmeyer internal docs style guide says one L